### PR TITLE
28 empty user profile

### LINKED ISSUE #28: Create empty user profile page with two sections
This task is the base task in which other tasks will be built on.

Acceptance criteria:

1. The left `about me` section should have the `mentor.me` logo with the gradient background colour as in the design. Nothing else is required to be added.

2. The right `information` section should have the `Dashboard` and `Edit Profile` links at the top right. Nothing else is also required to be added.

Screenshot 2020-09-15 at 6 49 59 PM

### PR DESCRIPTION
**Pull request by Nkoli**
Screenshot from 2020-09-24 20-22-57
Screenshot from 2020-09-24 20-23-11
Screenshot from 2020-09-24 20-23-25

Fixes #28

### CODE REVIEW

**Review by fola-ooo — APPROVED** [review]

File: src/App.css [hunk 1]
```diff
@@ -96,3 +93,47 @@
...
+
+.edit_profile_link {
+  color: #060553;
+  text-decoration-color: #00d4ff;
+  text-decoration-thickness: 2px;
+  margin-right: 90px;
+}
+
+.about_container {
+  height: 610px;
```
Comment (toystars) [inline src/App.css]: This class belongs to a div that takes full-screen height, yeah? So why does it have a height of `610px`?. If it is supposed to take full height, then a value of `100vh` will make it expand to the full browser window height.

File: src/App.css [hunk 2]
```diff
@@ -96,3 +93,47 @@
...
+    360deg,
+    #020024 11.39%,
+    rgba(8, 8, 115, 0.709638) 93.3%,
+    rgba(9, 9, 121, 0.685897) 100%
+  );
+}
+
+.about_container h1 {
+  position: absolute;
+  left: 3%;
```
Comment (toystars) [inline src/App.css]: I suppose this is the style from Figma, no? The `h1` element does not need to be positioned absolutely.

**Review by toystars — CHANGES_REQUESTED** [review]
@Nkoli I was expecting the styling of this page to be implemented using flex-box. The current implementation won't work. Please find some resources online on flex-box. We can have a short session if you require an in-person explanation on how it works.

File: src/App.css (hunk 1 above)
Comment (Nkoli) [reply src/App.css]: Yes it does, and thank you, i didn't know this. It works

File: src/App.css (hunk 2 above)
Comment (Nkoli) [reply src/App.css]: Yes, it's the style from figma. However, removing the absolute position changes the position of the text and just left aligns it without the margins or paddings and also does this stuff where it reduces the background height as well
Screenshot from 2020-09-23 17-22-35

File: src/App.css (hunk 1 above)
Comment (toystars) [reply src/App.css]: @Nkoli Did you check why the `100vh` works?

File: src/App.css (hunk 2 above)
Comment (toystars) [reply src/App.css]: @Nkoli you can use margins and paddings to achieve the same styling. Absolute positioning is great, but it is easy for it to get out of hand, so my approach is to use `position: absolute` for things like `Modals`, `Popovers`, `Tooltips`, `Dropdowns`, `DatePickers` etc

File: src/App.css (hunk 1 above)
Comment (Nkoli) [reply src/App.css]: Yes, it's not fixed-ish like using px is and represents a percentage of the height defined as the body height? So 100vh is equivalent to 100% of the initially defined body height. Do i understand this correctly?

File: src/App.css (hunk 2 above)
Comment (Nkoli) [reply src/App.css]: Okay, I'll use margins and paddings for this. I thought using absolute and then defining the percentage value for specific positioning allowed for better responsiveness than using margin and paddings for components that may be reused in different ways?

File: src/pages/userinformation/UserInformation.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  line-height: 60px;
+  font-weight: 500;
+}
+
+.dashboard_link {
+  align-items: right;
+  color: #060553;
+  text-decoration-color: #00d4ff;
+  text-decoration-thickness: 2px;
+  margin-right: 30px;
```
Comment (toystars) [inline src/pages/userinformation/UserInformation.css]: @Nkoli please convert these `px` values to `rem`

**Review by toystars — APPROVED** [review]
Looks good. Well done 👍🏾

**Pull request merged**